dialogue-core/src/main/java/com/palantir/dialogue/core/CautiousIncreaseAggressiveDecreaseConcurrencyLimiter.java

```diff
@@ -154,10 +158,22 @@ void onFailure(Throwable _throwable, PermitControl control) {
 
     interface PermitControl {
 
+        /**
+         * Indicates that the effect of the request corresponding to this permit on concurrency limits should be
+         * ignored.
+         */
         void ignore();
 
+        /**
+         * Indicates that the request corresponding to this permit was dropped and that the concurrency limit should be
+         * multiplicatively decreased.
+         */
         void dropped();
 
+        /**
+         * Indicates that the request corresponding to this permit was successful and that the concurrency limit should
+         * be additively increased.
+         */
```

Comment:
by putting these Javadoc on the interface instead of the class, they now show in IDE `F1` above